Prace_domowe/Praca_domowa1/Grupa1/Dawid_Janus/PD1_Dawid_Janus.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>Doceniam, że próbowano usuwać wartości skrajne mimo, że  nie było to wymagane w tej pracy domowej. Prawidłowo zastosowano metodę opartą o kwantyle. </p><p>Należy jednak pamiętać o tym, że generalnie z punktu widzenia ochrony przyrody najbardziej interesują nas największe pożary bo są najgroźniejsze - w tej sytuacji nie powinno się ich trwale usuwać z danych. Warto byłoby wspomnieć, że to operacja wykonana jedynie na potrzeby wizualizacji ale w dalszym modelowaniu nie byłaby stosowana.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>